packages/IMAPClient-Core.package/ICCommand.class/class/join..st

```diff
@@ -0,0 +1,6 @@
+commands
```

Comment:
Maybe a category named `utility` or `command building` would be more fit for this?